### PR TITLE
init

### PR DESCRIPTION
**Pull request by 06Pac**

### CODE REVIEW

**Review by NeuroCorgi — CHANGES_REQUESTED** [review]
requirements.txt is missing for a python environment.
PR description missing.

File: server/main.py
Comment (NeuroCorgi) [inline server/main.py:1]: Put tokens in ENV variables in your shell

**Review by EGheska — CHANGES_REQUESTED** [review]
cannot see anything regarding to the front end

**Review by EGheska — APPROVED** [review]

**Comment by jurijjzz** [comment]
Da normaljno, senior back ender

**Review by jurijjzz — APPROVED** [review]

**Review by jurijjzz — APPROVED** [review]

**Review by EGheska — APPROVED** [review]
Good code. Everything is understandable! BTW i am a front end'er so cannot understand anything

**Review by NeuroCorgi — APPROVED** [review]

**Review by NeuroCorgi — APPROVED** [review]

**Review by jurijjzz — APPROVED** [review]

**Review by EGheska — APPROVED** [review]

**Pull request merged**